### DISCUSSION TITLE
After computer start opensnitch not working

### DISCUSSION THREAD

**Discussion by devmalick** [evilsocket/opensnitch#848] [General]
OpenSnitch show disable on systemtry but the service systemd running, then i have to restart the service systemd then it works.
● opensnitchd.service - OpenSnitch is a GNU/Linux application firewall.
     Loaded: loaded (/usr/lib/systemd/system/opensnitchd.service; enabled; preset: disabled)
     Active: active (running) since Mon 2023-02-13 12:42:25 IST; 4min 7s ago
       Docs: https://github.com/evilsocket/opensnitch/wiki
   Main PID: 2785 (opensnitchd)
      Tasks: 16 (limit: 9034)
     Memory: 36.8M
        CPU: 4.907s
     CGroup: /system.slice/opensnitchd.service
             └─2785 /usr/bin/opensnitchd -rules-path /etc/opensnitchd/rules

**Comment by gustavo-iniguez-goya** [comment]
Hi @devmalick ,

Coul you provide information about your system? distribution, opensnitch version, `/var/log/opensnitchd.log`

If you're using Arch :point_right: #833

Thank you.
